cli/core/utils.go

```diff
@@ -197,7 +201,7 @@ func SortByStatus(validators map[string]Validator) ([]Validator, []Validator, []
 		return withdrawnValidators[i].Index < withdrawnValidators[j].Index
 	})
 
-	return inactiveValidators, activeValidators, withdrawnValidators
+	return awaitingActivationQueueValidators, inactiveValidators, activeValidators, withdrawnValidators
```

Comment:
if you're gonna add a new return group to this function signature, i would probably put it at the end right?

Reply:
i was going for "lifecycle" here, since the left-most validators would occur earliest and then progress left to right